presto-mysql/src/test/java/io/prestosql/plugin/mysql/BaseMySqlIntegrationSmokeTest.java

```diff
@@ -263,6 +263,9 @@ public void testColumnComment()
                 "SELECT column_name, comment FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = 'test_column_comment'",
                 "VALUES ('col1', 'test comment'), ('col2', null), ('col3', null)");
 
+        assertUpdate("COMMENT ON COLUMN tpch.test_column_comment.col1 IS 'test comment in col1'");
+        assertUpdate("COMMENT ON COLUMN tpch.test_column_comment.col1 IS ''");
```

Comment:
Please add test case for `IS NULL` and assertion to make sure exactly the column comment was set.

Reply:
@ebyhr  test case is added for IS NULL. Could you please provide me an example for the second part?

Reply:
We can add utility method for directly getting `ResultSet` from MySQL and then assert fields in `information_schema.columns`. 


```java
    private ResultSet executeQuery(String sql)
            throws SQLException
    {
        try (Connection connection = DriverManager.getConnection(mysqlServer.getJdbcUrl(), mysqlServer.getUsername(), mysqlServer.getPassword());
                Statement statement = connection.createStatement()) {
            return statement.executeQuery(sql);
        }
    }
```